package.json

```diff
@@ -37,8 +37,8 @@
     "docker:clean": "yarn docker:compose down --rmi all -v && rm -rf docker/wordpress/* docker/wordpress/.htaccess docker/wordpress-develop/* docker/logs/* docker/data/mysql/*",
     "lint": "eslint --ext .js --ext .jsx ./*.js _inc/client -c .eslintrc",
     "test-client": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js",
-    "add-textdomain": "grunt addtextdomain",
-    "build-pot": "grunt makepot",
+    "add-textdomain": "wpi18n addtextdomain --textdomain=jetpack --glob-pattern='!(docker|node_modules|tests|tools|vendor){*.php,**/*.php}'",
```

Comment:
`wpi18n@1.2.0` now supports `--exclude` for `addtextdomain` command so those ignored folders from glob pattern could be moved out.

That would be just for readability though, `glob-pattern` already works as expected.